doc/make/index.md

```diff
@@ -17,39 +17,27 @@ Platform-Specific Setup
 Generic Build Instructions
 --------------------------
 
-Setting up a basic release build:
+Setting up a basic parallelized release build:
 
 ```bash
-git clone https://github.com/leanprover/lean4 --recurse-submodules
+git clone https://github.com/leanprover/lean4
 cd lean4
-mkdir -p build/release
-cd build/release
-cmake ../..
-make
+cmake --preset release
+cmake --build --preset release -j$(nproc)
```

Comment:
This looks a lot harder to remember than `make` or `make -j4` which is what I usually use.

Reply:
Yes, I've simplified that step and adjusted the parallelism description. I still think it makes sense to give the instructions that give the fastest build for the majority of users first. It's just insane that neither make nor cmake provide a built-in way to do so.